### DISCUSSION TITLE
May Community Meeting

### DISCUSSION THREAD

**Discussion by Pellifer** [eclipse-symphony/symphony#243] [Announcements]
We are excited to host the first in-person community meeting for Project Symphony. The meeting will be held at the Microsoft Garage (Building 27) in Redmond, WA, on May the 20th , from 1pm-4pm. 
The first hour of the Community meeting will also be hosted on Teams and recorded to be shared on the Project Symphony YouTube Channel.  

**Agenda**
1:00 PM PT “Intro to Project Symphony and roadmap update”
2:00 PM PT “Meet the experts, Q&A, with opportunity to try hands-on-labs”
3:00 PM PT “User Experience and Usability study” 

Please sign up here: [Invite](https://forms.office.com/r/msGna9esiC)

We encourage community contributions and presentations so please submit your contributions to [Discord #Project Symphony](https://discord.com/channels/1195064864767496302/1195064864767496305) or send a mail to [symphony-dev@eclipse.org](mailto:symphony-dev@eclipse.org).

If you have any questions and suggestions regarding the Project Symphony or the community meeting, add your comment to the project forum [eclipse-symphony/symphony · Discussions · GitHub](https://github.com/eclipse-symphony/symphony/discussions).

**For more information on Project Symphony:** 
[Eclipse Symphony | projects.eclipse.org](https://projects.eclipse.org/projects/iot.symphony)
**For contributions and getting started guides:**
[eclipse-symphony/symphony: Symphony project (github.com)](https://github.com/eclipse-symphony/symphony)
[symphony/docs/README.md at main · eclipse-symphony/symphony (github.com)](https://github.com/eclipse-symphony/symphony/blob/main/docs/README.md)

**Comment by schuettecarsten** [comment]
Hi, I've registered using the form but did not recieve any kind of confirmation?

**Reply by Pellifer** [reply]
Hi, thanks or bring this to our attention. You should have got a "Pop up" msg that your registration was confirmed after submitting it. I will look at the settings on the MS Form tool we use to ensure it is setup in the right way. I can see that your registration was successful, looking forward to have your participation in this Community meeting.

**Comment by hariarun18** [comment]
Looking forward to this 🎉
